### PR TITLE
added http.pizza

### LINKED ISSUE #3: add kir-cat, kir-pizza
As a user, I would like to have the opportunity to play not only kir-kuty, but kir-cat and kir-pizza as well.
In these cases the images can come from http.cat and http.pizza (instead of http.dog).

-these new games should use the original code (add a parameter to the link?, that determines the mode?)
-place an icon for the new games in the main menu

### PR DESCRIPTION
**Pull request by FearsomeRover**

### CODE REVIEW

File: src/app/games/http/page.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export type Answers = {
     correctBtnIndex: number
 }
 
-function next() {}
+const animals = ['dog', 'pizza']
```
Comment (mozsarmate) [inline src/app/games/http/page.tsx]: -- Kid Bujdi, what is your favourite animal?
-- It is PIZZA!!!

File: src/app/games/http/page.tsx [hunk 2]
```diff
@@ -65,17 +66,29 @@ export default function HttpGame() {
...
-                    <div className='sidebar'>
-                        <p id='history'>Alma alva ring az ágon, alma álma áhitat, se érzés, se gondolat, tiszta fény az alma-álom.</p>
-                    </div>
-                    <ImageContainer errorCode={answers.correctHttpCode} />
-                    <div className='sidebar'>
-                        <p id='history'>Alma alva ring az ágon, alma álma áhitat, se érzés, se gondolat, tiszta fény az alma-álom.</p>
-                    </div>
-                </div>
+                {animalIdx == -1 ? (
+                    <>
```
Comment (mozsarmate) [inline src/app/games/http/page.tsx:129]: It would be nice to have a small piece of instructions here

**Review by mozsarmate — APPROVED** [review]
overall Very Nice!

I thought we 'll have a separate icon for each aNiMaL of the HTTP game on the Main Menu page. On the other hand, your solution is also nice. Implement the aboves if you feel so!
 (I like the hot swap button in the corner as well).

**Comment by FearsomeRover** [comment]
Haló ez nem volt a specifikációban!

**Pull request merged**